github/repos_rules.go

```diff
@@ -14,8 +14,10 @@ import (
 // BypassActor represents the bypass actors from a ruleset.
 type BypassActor struct {
 	ActorID *int64 `json:"actor_id,omitempty"`
-	// Possible values for ActorType are: Team, Integration
+	// Possible values for ActorType are: RepositoryRole, Team, Integration, OrganizationAdmin
 	ActorType *string `json:"actor_type,omitempty"`
+	// Possible values for BypassMode are: always, pull_request
```

Comment:
It seems odd to me that the original author noted that the possible values were completely different from these possible values. Are you sure that you found all the endpoints that the original author was covering?

Reply:
I believe so. Per my understanding of repo rules, you can create them on the org level and the repo level which should leave us with only the two endpoints. Both of these endpoints have the updated values. I may be missing something though so I'll take another look!

Reply:
I don’t see other references to the `Ruleset` struct in the codebase except for org rules which has the same possible values for the `BypassMode` field. I didn’t find anything else in the API docs either.

Reply:
OK, I've gone through the docs as well, and I think you are correct.  Thank you, @o-sama !